src/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,37 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getOrdersByUserId = async (raw_user_id) => {
+  const user_id = parseInt(raw_user_id, 10);
+
+  if (isNaN(user_id) || user_id < 1) {
+    throw new HttpError('The ID should be a number', 400);
```

Comment:
response code 400 is not documented in the swagger responses.

Reply:
<img width="1099" alt="image" src="https://user-images.githubusercontent.com/84173609/155416237-075dcf71-a4e1-4e23-8f30-41e57779c780.png">
👍 check please

and the I get all the orders and cancelled order showed null price, so I put order status so that we know why the price is null. 

Is that make sense? If I can improve in other ways, please let me know so that I can implement better code ☺️